pkg/util/cgroups/pid_mapper.go

```diff
@@ -107,6 +107,18 @@ func getPidMapper(procPath, cgroupRoot, baseController string, filter ReaderFilt
 	return pidMapper
 }
 
+// GetProcessNamespaceInode performs a stat() call on /proc/<pid>/ns/<namespace>
```

Comment:
I have another PR that will require this function in `system`. Ideally we'd want to keep in a single place, but we'll need to duplicate for now.
I think we should add a comment here and leave the original copy in `system`.